.travis.yml

```diff
@@ -25,7 +24,9 @@ matrix:
     fast_finish: true
     include:
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+          env: COMPOSER_FLAGS="--ignore-platform-reqs"
```

Comment:
is this because of the stream-filter library? if so i would move it to require-dev and suggest. if we support php 5.4, we should not fail with weird errors or tell people to ignore the platform and then they are surprised when trying to use streams.

for travis, we will need to keep the flag, as we do need the dev dependencies.